src/api/app/components/bs_request_overview_avatars_component.rb

```diff
@@ -13,14 +13,26 @@ def avatar_objects
     @avatar_objects ||= if @review.for_user?
                           [@review.user]
                         elsif @review.for_group?
-                          [@review.group.users, @review.group].flatten
+                          group_avatar_objects
                         elsif @review.for_package?
-                          @review.package.project.users
+                          package_avatar_objects
```

Comment:
I had to extract part of the code to methods to avoid high cyclomatic complexity.